ParisBNF/abb/BNFabb51.xml

```diff
@@ -29,7 +30,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <collection>Manuscrits orientaux</collection>
                         <collection>Fonds éthiopien</collection>
                         <collection>d'Abbadie</collection>
-                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100878602.r=Abbadie%2051?rk=64378;0">BnF Éthiopien d'Abbadie 51</idno>
+                        <idno facs="https://gallica.bnf.fr/view3if/ga/ark:/12148/btv1b100878602">BnF Éthiopien d'Abbadie 51</idno>
```

Comment:
i had corrected my comment - the link was correct only too long
should be https://gallica.bnf.fr/ark:/12148/btv1b100878602

Reply:
no problem, I did; but how will I get the right link? If I click on the symbol with letter iiif, I get exactly the link that I have removed, the same but with an extension in the middle (view...), https://gallica.bnf.fr/view3if/ga/ark:/12148/btv1b100878602

Reply:
could you please merge, if there are no further requests?
Thanks!

Reply:
for BNF do not click on iiif, it was encoded differently, just as the guidelines say